components/renku_data_services/authz_migrations/versions/748ed0f3439f_create_project_user_permissions.py

```diff
@@ -0,0 +1,51 @@
+"""create project-user permissions
+
+Revision ID: 748ed0f3439f
+Revises:
+Create Date: 2023-11-13 13:59:12.707850
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "748ed0f3439f"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "project_user_authz",
+        sa.Column("project_id", sa.String(length=36), nullable=False),
+        sa.Column("role", sa.Integer(), nullable=False),
+        sa.Column("user_id", sa.String(length=36), nullable=True),
```

Comment:
Yeah this is autogenerated migration from alembic based on the orm schema. Once we update the thing above it will match. But as I said above keycloak user ids are uuid4 and uuid4 are 36 characters.